spec/hbw/features/tasks/task_list_with_filter_bp_mock.yml

```diff
@@ -0,0 +1,173 @@
+          processDefinitionId: "testProcess:469:222222"
+        - <<: *default_task
+          id: "111113"
+          name: "Other assigned task"
+          due: "2017-07-30T12:07:59.000+03:00"
+          formKey: "test_form_with_disabled_buttons.yml"
+          processInstanceId: 222221
+          description: 'Some test description'
+          processDefinitionId: "testProcess:469:222221"
+    - params: "active=true&candidateUser=demo&processVariables[][name]=homsOrderCode&processVariables[][operator]=like&processVariables[][value]=%"
+      response:
+        - <<: *default_task
+          id: "111112"
+          name: "Unassigned task"
+          due: "2016-07-30T12:07:59.000+03:00"
+          priority: 25
+          processInstanceId: 222222
+          processDefinitionId: "testProcess:469:222222"
+          assignee: null
+        - <<: *default_task
+          id: "111111"
+          name: "Other unassigned task"
+          due: "2017-07-30T12:07:59.000+03:00"
+          formKey: "test_form_with_disabled_buttons.yml"
+          processInstanceId: 222221
+          description: 'Some test description'
+          processDefinitionId: "testProcess:469:222221"
+          assignee: null
+
+  process-instance:
+    - params: "active=true&variables[][name]=homsOrderCode&variables[][value]=ORD-1&variables[][operator]=eq"
+      response:
+        default_data: &default_process-instances
+          id: "111111"
+          businessKey: null
+          caseInstanceId: null
+          suspended: false
+          ended: false
+          processDefinitionId: "testProcess:1:1"
+          tenantId: null
+        data:
+          - <<: *default_process-instances
+            id: "111111"
+            processDefinitionId: "testProcess:469:222221"
+
+  task/111111/submit-form:
+    - params: 'variables[FL_EmptyFileUpload][value]=[]&variables[FL_MultipleFileUpload][value]=[{"origin_name":"logo.svg","field_name":"multipleFileUpload","end_point":"http://127.0.0.1:9000","bucket":"bucket-name"},{"origin_name":"file_with_long_name.pdf","field_name":"multipleFileUpload","end_point":"http://127.0.0.1:9000","bucket":"bucket-name"}]&variables[FL_SingleFileUpload][value]=[{"name":"file3.txt+(21.07.2021+16:57:16)","origin_name":"file3.txt","field_name":"multipleFileUpload","end_point":"http://127.0.0.1:9000","bucket":"bucket-name"},{"name":"file4.txt+(21.07.2021+16:57:16)","origin_name":"file4.txt","field_name":"multipleFileUpload","end_point":"http://127.0.0.1:9000","bucket":"bucket-name"},{"origin_name":"file.txt","field_name":"singleFileUpload","end_point":"http://127.0.0.1:9000","bucket":"bucket-name"}]&variables[homsOrderDataFileList][value]=[{"origin_name":"file2.txt","field_name":"paramAbsentFileUpload","end_point":"http://127.0.0.1:9000","bucket":"bucket-name"}]'
+      response: [""]
+
+  task/111111/claim:
+    - params: "userId=demo"
+      response: null
+
+  task/111112/claim:
+    - params: "userId=demo"
+      response: null
```

Comment:
can't see any submit or claim in the test

Reply:
Removed all unnecessary